duckduckgo/macos-browser
# Do not cancel detection if it is related to developer redirect

Task/Issue URL: https://app.asana.com/0/856498667320406/1207772445672485/f

**Description**:

Fix attribution detection that was failing due to developer redirect (cancellation and reloading) of the request by explicitly ignoring failure events in such case.

**Steps to test this PR**:

*Tests validation:*

You can revert the BSK change, tests should fail.

*Live validation*

You need UK VPN and search location set to UK.

- Search for ‘floral dress’
- Click on product ad for joebrowns.co.uk - shopping vertical, not the text ad!
- Validate that in the privacy dashboard, site has been correctly attributed.



**Definition of Done**:


---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
